### ISSUE TITLE
[INSTALL, Dockerfile] ply 3.3 not anymore in the dependency repository

### ISSUE THREAD

**Issue by folkvir** [anapsid/anapsid#2]
ply 3.3 is not anymore in the dependency repository of pip. There are only version 3.4 and newer versions. The only way to get it work is to manually download the dependency and install it by hand.
* ply repo link: http://www.dabeaz.com/ply/
* choose the 3.3, install and rerun the installation setup of anapsid

or use this Dockerfile
```Dockerfile

FROM python:2.7
WORKDIR /app
RUN wget http://www.dabeaz.com/ply/ply-3.3.tar.gz
RUN pip install ./ply-3.3.tar.gz
COPY . /app
RUN python /app/setup.py install
```
```bash
# build the docker file:
docker build . -t anapsid
# run the dockerfile in bash mode: 
docker run -it anapsid bash
# then run the example inside the container: 
python scripts/run_anapsid -e example/endpoints/dbpediaEndpoint -q example/dbpediaQueries/example.sparql -p b -s False -o False -d SSGM -a True -w False -r False
```

**Comment by Lars-H** [comment]
Alternatively, you can get the required PLY version (3.3) from the [official page](http://www.dabeaz.com/ply/), untar it and use the setup.py to install it.

**Comment by Lars-H** [comment]
Alternatively, you can get the required PLY version (3.3) from the [official page](http://www.dabeaz.com/ply/), untar it and use the setup.py to install it.

**Comment by folkvir** [comment]
Of course. This is what I did in the dockerfile. Download the tar, and install it using pip. But you can do it without the Dockerfile :smiley:
